content/main.js

```diff
@@ -390,18 +390,25 @@ var tblatex = {
       div.parentNode.removeChild(div);
   }
 
-  function replace(string, pattern, replacement) {
+  function replace_marker(string, replacement) {
+    var marker = "__REPLACE_ME__";
+    var oldmarker = "__REPLACEME__";
+    var len = marker.length;
     var log = "";
-    var i = string.indexOf(pattern);
+    var i = string.indexOf(marker);
     if (i < 0) {
-      log += "\n!!! Could not find the place marker '__REPLACE_ME__' in your template.\n";
-      log += "This would be the place, where your LaTeX expression is inserted.\n";
-      log += "Please edit your template and add this marker.\n";
-      return [, log];
+      // Look for old marker
+      i = string.indexOf(oldmarker);
+      if (i < 0) {
+        log += "\n!!! Could not find the place marker '" + marker + "' in your template.\n";
+        log += "This would be the place, where your LaTeX expression is inserted.\n";
+        log += "Please edit your template and add this marker.\n";
+        return [, log];
+      } else {
+        len = oldmarker.length;
```

Comment:
~I think you're missing a closing brace here, no?~

~oh I got confused by the indentation~

no actually I'm still confused

Reply:
No need to be confused! You spotted it absolutely corrected: A closing `}` is missing. Sorry and thanks for spotting it!!!